File: EXAMPLES/IDE/python/banking-app.py
import os

def clear_screen():
    os.system('cls')
clear_screen()


currentBalance = 1000
openingBalance = 1000
previousBalance = 0

# Displaying a selection menu for a banking app
print("Welcome to SEAP-10 Banking App!")
print("1. Check Balance")
print("2. Deposit Money")
print("3. Withdraw Money")
print("4. Exit")

# Get user input for selection
choice = input("Please enter your choice (1-4): ")

# user's selection
if choice == "1":
    print(f"Your current balance is {currentBalance}")
elif choice == "2":
    amount = float(input("Enter the amount to deposit: "))
    currentBalance += amount
    print(f"${amount} deposited successfully.")
    # Update balance logic here
elif choice == "3":
    amount = float(input("Enter the amount to withdraw: "))
    if amount <= currentBalance:
        currentBalance -= amount
        print(f"${amount} withdrawn successfully.")
    else:
        print("Insufficient funds.")
    # Update balance logic here
elif choice == "4":
    print("Thank you for using our banking app. Goodbye!")
else:
    print("Invalid choice. Please select a valid option.")

File: EXAMPLES/IDE/java/banking-app.java
import java.util.Scanner;

public class BankingApp {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        // Displaying a selection menu for a banking app
        System.out.println("Welcome to Our Banking App!");
        System.out.println("1. Check Balance");
        System.out.println("2. Deposit Money");
        System.out.println("3. Withdraw Money");
        System.out.println("4. Exit");
        
        // Get user input for selection
        System.out.print("Please enter your choice (1-4): ");
        int choice = input.nextInt();
        
        // Process the user's selection
        switch (choice) {
            case 1:
                System.out.println("Your current balance is 1000");
                break;
            case 2:
                System.out.print("Enter the amount to deposit: ");
                double amount = input.nextDouble();
                System.out.println("$" + amount + " deposited successfully.");
                // Update balance logic here
                break;
            case 3:
                System.out.print("Enter the amount to withdraw: ");
                double withdrawAmount = input.nextDouble();
                System.out.println("$" + withdrawAmount + " withdrawn successfully.");
                // Update balance logic here
                break;
            case 4:
                System.out.println("Thank you for using our banking app. Goodbye!");
                break;
            default:
                System.out.println("Invalid choice. Please select a valid option.");
        }
    }
}